website/docs/cairo-1/05-testing/01-unit-testing.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_label: Unit testing
+---
+
+# Unit testing
+
+
+## Writing your first test (TODO test if the example works)
+
+First, add to the file `src/lib.cairo` following code:
+```
+fn sum(a: felt252, b: felt252) -> felt252 {
+    return a + b;
+}
+```
+
+Now, let's test this function. Create a file `tests/test_sum.cairo`:
+```
+use your_project_name::sum
+
+#[test]
+fn test_sum() {
+    assert(sum(2, 3) == 5, 'sum incorrect');
+}
+```
+
+Now run your test using command:
+```
+protostar test-cairo1 ./tests
+```
+
+You should see something like this:
+```
+Collected 2 suites, and 3 test cases (10.64)
+[PASS] tests/test_sum.cairo Then (time=0.00s)
+Test suites: 1 passed, 1 total
+Tests:       1 passed, 1 total
+Seed:        2752673895
+17:20:43 [INFO] Execution time: 5.0 s
+```
+
+## Test collecting
+Protostar collects all test suites specified under the path passed as an argument. You can pass either directory or a specific file. Test suite is every `.cairo` file with name starting from `test_` or ending with `_test`. Protostar considers each function, inside a test suite, with `#[test]` attribute as a test case.
```

Comment:
```suggestion
Protostar collects all test suites specified under the path passed as an argument. You can pass either directory or a specific file. A test suite is every `.cairo` file with a name starting from `test_` or ending with `_test`. Protostar considers each function, inside a test suite, with the `#[test]` attribute as a test case.
```

Reply:
> the `#[test]` 
I think we shouldn't add the here it is kinda name